src/lib/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import PaginationItem from './PaginationItem';
+
+const Pagination = props => (
+  <footer className="p-pagination">
+    { props.children }
+  </footer>
```

Comment:
Not sure we should use `footer` we can't be sure this is only footer on the page or in an `section`.